packages/docs/src/routes/Axes.tsx

```diff
@@ -5,8 +5,16 @@ import javascript from 'highlight.js/lib/languages/javascript'
 import { CodeSnippet } from '@carbon/react/es'
 import PageHeader from '../components/PageHeader'
 import ConfigExamples from '../components/ConfigExamples'
+import styled from 'styled-components'
 
-import './Axes.scss'
+// const ExampleImage = styled.img`
+// 	width: 100%;
+// `
```

Comment:
do we need this?

Reply:
It's for the commented content that was omitted in order to not conflict with the design folks. I recommend keeping this in case we ever bring the content below back.